Adjust the oc edit procedures (#477)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/con_primary-parameters-of-the-servicetelemetry-object.adoc

Use the `backends` parameter to control which storage back ends are available for storage of metrics and events, and to control the enablement of Smart Gateways that the `clouds` parameter defines. For more information, see xref:clouds_assembly-installing-the-core-components-of-stf[].

> Currently, you can use Prometheus as the metrics storage back end and Elasticsearch as the events storage back end. The Service Telemetry Operator orchestrates the creation of other custom resource objects that the Prometheus Operator and Elastic Cloud on Kubernetes Operator watches and results in the creation of Prometheus and Elasticsearch workloads.

Reviewer: ```suggestion
You can use Prometheus as the metrics storage back end and Elasticsearch as the events storage back end. You can use the Service Telemetry Operator to create other custom resource objects that the Prometheus Operator and Elastic Cloud on Kubernetes Operator watch to create Prometheus and Elasticsearch workloads.
```

.Procedure

. Edit the `ServiceTelemetry` object:
+
[source,bash]
----
$ oc edit stf default
----

> . Configure the backends.metrics.prometheus.enabled parameter to `true`:

Reviewer: ```suggestion
. Set the value of the backends.metrics.prometheus.enabled parameter to `true`:
```

standard-csi         cinder.csi.openstack.org   Delete          WaitForFirstConsumer   true                   20h
----

. Edit the `ServiceTelemetry` object:
+
[source,bash]
----
$ oc edit stf default
----

> . Configure the backends.metrics.prometheus.storage parameters to configure and enable persistent storage:

Reviewer: ```suggestion
. Set the value of the backends.metrics.prometheus.enabled parameter to `true` and the value of backends.metrics.prometheus.storage.strategy to `persistent`
```

.Procedure

. Edit the `ServiceTelemetry` object:
+
[source,bash]
----
$ oc edit stf default
----

> . Configure the backends.events.elasticsearch.enabled parameter to `true`:

Reviewer: ```suggestion
. Set the value of the backends.events.elasticsearch.enabled parameter to `true`:
```

standard-csi         cinder.csi.openstack.org   Delete          WaitForFirstConsumer   true                   20h
----

. Edit the `ServiceTelemetry` object:
+
[source,bash]
----
$ oc edit stf default
----

> . Configure the backends.events.elasticsearch.storage parameters to configure and enable persistent storage:

Reviewer: ```suggestion
. Set the value of the backends.events.elasticsearch.enabled parameter to `true` and the value of backends.events.elasticsearch.storage.strategy to `persistent`:
```
